ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -32,6 +31,7 @@ import (
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config/ami"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/docker/libcompose/project"
+	"github.com/sirupsen/logrus"
```

Comment:
Do you know why it appears that logrus has been added? we have logrus statements in this file that predate this PR